### DISCUSSION TITLE
sandbox-Karl-June13/v10/management-center/concepts/cmc_manageorg

### DISCUSSION THREAD

**Discussion by giscus** [HCL-TECH-SOFTWARE/HCLSoftware-Documentation-Home#92] [Q&A]
# sandbox-Karl-June13/v10/management-center/concepts/cmc_manageorg

You can manage member groups, organizations, users, and approval through the Management Center.

http://10.190.67.78/sandbox-Karl-June13/v10/management-center/concepts/cmc_manageorg.html

**Comment by ChristineNiro** [comment]
Test

**Comment by ChristineNiro** [comment]
Successful test.
